parity-crypto/Cargo.toml

```diff
@@ -24,9 +27,15 @@ aes = "0.3.2"
 aes-ctr = "0.3.0"
 block-modes = "0.3.3"
 pbkdf2 = "0.3.0"
+rand = "0.6"
+rustc-hex = "2.0"
 subtle = "2.1"
 zeroize = "0.9.1"
 
 [dev-dependencies]
 criterion = "0.2"
 hex-literal = "0.2"
+
+[features]
+default = []
+publickey = ["eth-secp256k1", "lazy_static", "ethereum-types"]
```

Comment:
Does it make sense to name this feature `secp256k1`? `publickey` seems too general

Reply:
I agree in general, but this exact name (secp256k1) seems not the best option, because we already have eth-secp256k1 feature and it will be two similar names. Naming is the hardest part of coding :-(